src/sentry/integrations/github/webhook.py

```diff
@@ -61,18 +68,25 @@ def __call__(self, event):
                     repo.config['name'] = event['repository']['full_name']
                     repo.save()
 
-                self._handle(event, orgs[repo.organization_id], repo)
+                self._handle(event, orgs[repo.organization_id], repo, host)
 
 
 class InstallationEventWebhook(Webhook):
     # https://developer.github.com/v3/activity/events/types/#installationevent
-    def __call__(self, event):
+    def __call__(self, event, host=None):
         installation = event['installation']
         if installation and event['action'] == 'deleted':
-            integration = Integration.objects.get(
-                external_id=installation['id'],
-                provider=self.provider,
-            )
+            if host:
+                external_id = '{}:{}'.format(host, event['installation']['id'])
+                integration = Integration.objects.get(
+                    external_id=external_id,
+                    provider=self.provider,
+                )
+            else:
+                integration = Integration.objects.get(
+                    external_id=event['installation']['id'],
+                    provider=self.provider,
+                )
```

Comment:
No need to duplicate the `Integration.objects.get` block, this could be written

```py
external_id = event['installation']['id']
if host:
    external_id = '{}:{}'.format(host, event['installation']['id'])

integration = Integration.objects.get(
    external_id=external_id,
    provider=self.provider,
)
```

Reply:
I saw you changed one in 47fffd4, any reason you can't change this one here too?